app/templating/summary_context.py

```diff
@@ -14,16 +14,11 @@ def build_summary_rendering_context(schema_json, answer_store, metadata):
     """
     navigator = PathFinder(schema_json, answer_store, metadata)
     path = navigator.get_routing_path()
-    sections = []
+    groups = []
+
     for group in schema_json['groups']:
-        for block in group['blocks']:
-            if block['id'] in [location.block_id for location in path] and block['type'] == 'Questionnaire':
-                link = url_for('questionnaire.get_block',
-                               eq_id=metadata['eq_id'],
-                               form_type=metadata['form_type'],
-                               collection_id=metadata['collection_exercise_sid'],
-                               group_id=group['id'],
-                               group_instance=0,
-                               block_id=block['id'])
-                sections.extend([Section(section, answer_store.map(), link) for section in block['sections']])
-    return sections
+        if 'title' in group and group['title'] not in ('Introduction', 'Summary') \
```

Comment:
There is no type to latch onto for the group. So any change here would require either looking at a block within the group (which could be problematic if there are multiple blocks in the group) or introducing some other property to identify that it should not be displayed in the summary (which may make more sense). Feel free to change this if it makes sense, I am not in for 3 weeks and won't get an opportunity to work on this, sorry.